### PR TITLE
Added contacts to JLab VO/move towards oasis usage

### PR DESCRIPTION
**Pull request by T-Britton**
@efajardo has been working with me on enabling other projects at JLab to use cvmfs (Oasis).  This PR also adds a couple contacts as requested

### CODE REVIEW

**Comment by matyasselmeci** [comment]
If you want to use OASIS, you also have to list some managers with the DNs of their certs. For example, this is from LIGO.yaml:
```
OASIS:
  Managers:
    - Name: Brian Bockelman
      DNs: /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=bbockelm/CN=659869/CN=Brian
        Paul Bockelman
      ID: 66ee5cfb622a7343dac85dee42815d1f4fbc2d85
    - Name: James Alexander Clark
      DNs: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=James Alexander
        Clark 4038
      ID: 504894b714071a10a647aaaa4e1fe7ee3186430b
```
Edgar, do you know if you need an OASISRepoURLs list as well?

**Comment by T-Britton** [comment]
good to know.  Thanks. I'll modify the yaml when I confirm the managers

File: virtual-organizations/JLab.yaml [hunk 1]
```diff
@@ -23,7 +38,7 @@ FieldsOfScience:
 ID: 99
 LongName: Jefferson Lab
 OASIS:
-  UseOASIS: false
+  UseOASIS: true
```
Comment (efajardo) [inline virtual-organizations/JLab.yaml:41]:
```suggestion
  OASISRepoURLs:
  - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/ligo.osgstorage.org
  - http://binaries.ligo.uwm.edu:8000/cvmfs/ligo-containers.opensciencegrid.org
  - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/gwosc.osgstorage.org
  UseOASIS: true
```

File: virtual-organizations/JLab.yaml [hunk 2]
```diff
@@ -23,7 +38,11 @@ FieldsOfScience:
 ID: 99
 LongName: Jefferson Lab
 OASIS:
-  UseOASIS: false
+  OASISRepoURLs:
+  - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/ligo.osgstorage.org
+  - http://binaries.ligo.uwm.edu:8000/cvmfs/ligo-containers.opensciencegrid.org
+  - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/gwosc.osgstorage.org
```
Comment (efajardo) [inline virtual-organizations/JLab.yaml:44]:
```suggestion
  OASISRepoURLs:
  - http://hcc-cvmfs-repo.unl.edu:8000/cvmfs/jlab.osgstorage.org
```

**Pull request merged**